modules/k8s-cluster/main.tf

```diff
@@ -73,7 +73,7 @@ resource "aws_cloudformation_stack" "worker-nodes" {
     NodeAutoScalingGroupDesiredCapacity = "0"
     NodeAutoScalingGroupMaxSize         = "0"
     NodeInstanceType                    = "t3.medium"
-    NodeVolumeSize                      = "40"
+    NodeVolumeSize                      = "120"
```

Comment:
I think this is the legacy worker node group

Reply:
🙈